src/store/modules/dashboard/getters.ts

```diff
@@ -1,2 +1,22 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import type { Getter } from 'vuex';
+
+import type { DashboardState, ScopeType } from '@/store/modules/dashboard/type';
+
+export const getItems: Getter<DashboardState, any> = (state): any => (type: ScopeType, filters, viewers) => {
+    let result = state[`${type.toLocaleLowerCase()}Items`];
+    if (viewers && viewers !== 'ALL') {
+        result = result?.filter((d) => d.viewers === viewers);
+    }
+    filters.forEach((d) => {
+        if (d.k === 'label' && Array.isArray(d.v)) {
+            d.v.forEach((value) => {
+                result = result.filter((item) => item.labels.includes(value));
+            });
+        }
+        if (!d.k && d.v) {
+            result = result.filter((item) => item.name.includes(d.v));
+        }
+    });
+    return result;
+};
```

Comment:
suggestion: What about add `workspaceDashboardList` and `projectDashboardList` to getters?

This `getItems` getter returns a function only one time. 
It does not execute getter callback reactively to state, because the context state is used is not the getter's context. Reactive state is used in the context inside the returned function.
So, there is no reason to use getter. It's gonna work the same even if it is just a pure helper function.